adding_species_manuscript.tex

```diff
@@ -453,34 +421,65 @@ \subsection*{Additions to \stdpopsim}
 a nematode (\textit{Caenorhabditis elegans}),
 two flowering plants including a crop (\textit{Helianthus annuus}),
 an algae (\textit{Chlamydomonas reinhardtii}),
-and two bacteria,
-in addition to four primates 
-and a common mammalian associate of primates (\textit{Canis familiaris}). % Elise: An aside, I find this way of describing dogs to be hilariously academic
-Not all of these have genetic maps or demographic models,
+two bacteria,
+%in addition to
+four primates and a common mammalian associate of primates (\textit{Canis familiaris}).
+% Elise: An aside, I find this way of describing dogs to be hilariously academic
+% Ilan: I concur :-)
+Not all of these have genetic maps or demographic models (see Figure \ref{fig:tree}),
 but this lays the framework for future contributions.
+\begin{figure}
+    \includegraphics[width=\linewidth]{figs/species_fig}
+	\caption{Phylogenetic tree of species available in the \stdpopsim catalog,
+		including the six species we published in the original release \citep[in blue]{Adrion2020}, and 15 species that have since been added (in orange).
+		Solid circles indicate species that have one (light grey) or more
+		(dark grey) demographic models and genetic maps.
+	\label{fig:tree} }
+\end{figure}
 
-A key feature added to \stdpopsim that expands the diversity of species that can
-be realistically modeled is the inclusion of recombination by gene conversion,
-which is essential for organisms such as \textit{E.\ coli}
-that lack crossing over.
-\colorbox{yellow}{[ TODO: summarize model of GC (see PR\#57) ]}
+Expanding the species catalog required adding several functionalities to the simulation framework.
+We thus upgraded the neutral simulation engine, \texttt{msprime}, from version 0.7.4 to version 1.0. \citep{Baumdicker2022}.
+This upgrade provides a number of benefits such as a discrete site model of mutation,
+so that simulated data will now %, as in real data,
+have a small proportion of sites with multiple mutations and possibly more than two alleles.
+Another key feature enabled by this upgrade was recombination by gene conversion,
+which is essential for modeling genomes of bacteria and archea.
```

Comment:
My understanding is that what bacteria do acts like gene conversion (and so is implemented using the "gene conversion" widgets of the simulators) but is called something different (e.g. "horizontal transfer"), so this ought to be readjusted a bit?

Reply:
I'm editing this now. My plan is to start by discussing "non-crossover recombination" without leaning on any specific terminology (such as GC or HGT). I'm adding citation to several studies on recombination in bacteria and archea